lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -138,11 +137,9 @@ def generate_log_for_all_tags
 
       log = generate_unreleased_section
 
```

Comment:
This is intentionally more explicit, than trying to iterate through the `@filtered_tags` and create sections from that array.